eng/common/scripts/Service-Level-Readme-Automation.ps1

```diff
@@ -0,0 +1,255 @@
+<#
+.SYNOPSIS
+Update unified ToC file for publishing reference docs on docs.microsoft.com
+
+.DESCRIPTION
+Given a doc repo location and a location to output the ToC generate a Unified
+Table of Contents:
+
+* Get list of packages onboarded to docs.microsoft.com (domain specific)
+* Get metadata for onboarded packages from metadata CSV
+* Build a sorted list of services
+* Add ToC nodes for the service
+* Add "Core" packages to the bottom of the ToC under "Other"
+
+ToC node layout:
+* Service (service level overview page)
+  * Client Package 1 (package level overview page)
+  * Client Package 2 (package level overview page)
+  ...
+  * Management
+    * Management Package 1
+    * Management Package 2
+    ...
+
+.PARAMETER DocRepoLocation
+Location of the documentation repo. This repo may be sparsely checked out
+depending on the requirements for the domain
+
+.PARAMETER TenantId
+The aad tenant id/object id for ms.author.
+
+.PARAMETER ClientId
+The add client id/application id for ms.author.
+
+.PARAMETER ClientSecret
+The client secret of add app for ms.author.
+#>
+
+param(
+  [Parameter(Mandatory = $true)]
+  [string] $DocRepoLocation,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TenantId,
+
+  [Parameter(Mandatory = $false)]
+  [string]$ClientId,
+
+  [Parameter(Mandatory = $false)]
+  [string]$ClientSecret
+)
+. $PSScriptRoot/common.ps1
+. $PSScriptRoot/Helpers/Metadata-Helpers.ps1
+
+Set-StrictMode -Version 3
+
+function GetClientPackageNode($clientPackage) {
+  $packageInfo = &$GetDocsMsTocDataFn `
```

Comment:
We should probably check to make sure this function exists otherwise this will fail in a strange way.

Reply:
It is being removed.